configure.ml

```diff
@@ -936,7 +936,7 @@ let config_runtime () =
   | Some flags -> string_split ',' flags
   | _ when use_custom -> [custom_flag]
   | _ when !prefs.local ->
-    ["-dllib";"-lcoqrun";"-dllpath";coqtop/"kernel/byterun"]
+    ["-dllib";"-lcoqrun";"-dllpath";("\"" ^ coqtop ^ "/kernel/byterun\"")]
```

Comment:
Not an expert here, but you could maybe use `String.escape`?

Reply:
I didn't know about this function, but I don't think it would help here, since `String.escaped` does not escape spaces/enclose the string in literal double quotes. These literal quotes are needed to make sure that the printed dllpath appearing in `config/Makefile` is properly escaped.

Reply:
Would Filename.quote work here?